src/azure-cli/azure/cli/command_modules/reservations/_params.py

```diff
@@ -31,3 +77,32 @@ def load_arguments(self, _):
 
     with self.argument_context('reservations catalog show') as c:
         c.argument('reserved_resource_type', arg_type=get_enum_type(ReservedResourceType))
+
+    with self.argument_context('reservations reservation-order calculate') as c:
+        c.argument('billing_scope_id', billing_scope_id_type)
+        c.argument('instance_flexibility', instance_flexibility_type)
+        c.argument('renew', renew_type)
+        c.argument('reserved_resource_type', reservation_resource_type_param_type)
+        c.argument('applied_scope_type', applied_scope_type_param_type)
+        c.argument('billing_plan', billing_plan_type)
+        c.argument('term', term_type)
+        c.argument('display_name', display_name_type)
+        c.argument('sku', sku_type)
+        c.argument('applied_scope', applied_scope_param_type)
+        c.argument('quantity', quantity_type)
+        c.argument('location', options_list=['--location'], help='Values from: `az account list-locations`.')
+
+    with self.argument_context('reservations reservation-order purchase') as c:
+        c.argument('reservation_order_id', help='Id of reservation order to purchase, generate by az reservations reservation-order calculate.')
```

Comment:
Please add quote for az command.
What about using `..., which is generated by 'az ...'`?

Reply:
make sense